src/main/scala/nl.knaw.dans.easy.deposit/Mailer.scala

```diff
@@ -64,6 +65,7 @@ case class Mailer(smtpHost: String,
       put("datasetTitle", data.title)
       put("myDatasetsUrl", myDatasets.toString)
       put("doi", data.doi)
+      put("hasFilesXml", nrOf(attachments) > 2)
```

Comment:
Please, look for "files.xml" in the attachments, instead of deriving this from the number of attachments.